dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIT.java

```diff
@@ -456,6 +457,51 @@ private static Condition getRefererCondition(final String referer) {
                 .build();
     }
 
+    /**
+     * Method to test: {@link ExperimentsAPIImpl#getResults(Experiment)}
+     * When: get the Experiment's Results
+     * Should: get the Split traffic too
+     */
+    @Test
+    public void getSplitTrafficInsiExperimentResults()
```

Comment:
isn't this a typo `getSplitTrafficInsiExperimentResults`
shouldn't it be `getSplitTrafficInsideExperimentResults` ?

Reply:
done https://github.com/dotCMS/core/pull/24826/commits/74a4521604f4260e233a96e2e407d51841984bb5#diff-25dea0f580007d55e9704544452fcf0833029a2294fb814f4dddfb1bb2e4ba9bR466